mods/default/furnace.lua

```diff
@@ -292,6 +292,7 @@ minetest.register_node("default:furnace", {
 	on_construct = function(pos)
 		local meta = minetest.get_meta(pos)
 		meta:set_string("formspec", default.get_furnace_inactive_formspec())
+		meta:set_string("infotext", 'Furnace inactive\n(Item: Empty; Fuel: Empty)')
```

Comment:
`furnace_node_timer(pos, 0)` for both lines would probably do the same, without code duplication.

Reply:
It gives an error if I change it to that:
```lua
...
local meta = minetest.get_meta(pos)
furnace_node_timer(pos, 0)
local inv = meta:get_inventory()
...
```
Am I doing it wrong?

Reply:
Call it after the `set_size` calls below.

Reply:
Still gives an error:
```
2019-09-13 22:21:41: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'default' in callback item_OnPlace(): Runtime error from mod 'default' in callback node_on_construct(): .../usr/bin/../games/minetest_game/mods/default/furnace.lua:203: attempt to index local 'srclist' (a nil value)
```

Reply:
Done! Check the latest commit.